src/components/ProjectContent.jsx

```diff
@@ -0,0 +1,60 @@
+import projectimg from '../assets/project.jpg';
+
+function ProjectContent() {
+  return (
+    <div className="grid grid-cols-1 sm:grid-cols-1 md:grid-cols-2 gap-8  p-3">
+      <div className="flex flex-col p-3 ">
+        <img
+          src={projectimg}
+          alt="project"
+          className="sm:h-48 sm:w-auto md:w-auto lg:w-80  rounded"
+        />
+        <h1 className="text-center sm:text-center md:text-left lg:text-left text-2xl text-blue-500 py-3 font-normal">
+          Project Name
+        </h1>
+        <span className="text-sm text-center sm:text-center md:text-left lg:text-left">
+          Mar 12 2022
+        </span>
+        <span className="text-center sm:text-center md:text-left lg:text-left pb-4">
+          Part-Time
+        </span>
+        <p className="flex flex-col items-center lg:flex md:flex">
+          You can load project details such as the project name, classification,
+          and type, as well as the start and end dates, to existing and new
+          projects. You load project details to existing projects by using a
+          data file with the same format as the ProjectDetails.csv data load
+          file
+          <br />
+          <br />
+          You load project details to existing projects by using a data file
+          with the same format as the ProjectDetails.csv data load file

```

Comment:
The details of the project should be a prop as well.

Reply:
done